contracts/mocks/ReentrancyExploiter.sol

```diff
@@ -73,7 +47,9 @@ contract ReentrancyExploiter {
   }
 
   receive() external payable {
-    execute(_fallbackTargets, _fallbackValues, _fallbackData);
+    if (!hasReentered) {
+      hasReentered = true;
+      execute(target, value, data);
```

Comment:
`hasReentered` needs to be cleared after the execution otherwise the contract cannot be used more than once.

Reply:
good catch